lib/ui/painting.dart

```diff
@@ -1568,6 +1568,15 @@ class Paint {
   }
 }
 
+/// The color space that an [Image] uses.
+enum ColorSpace {
+  /// The sRGB color gamut.
+  sRGB,
+  /// A color space that is backwards compatible with sRGB but can represent
+  /// colors outside of that gamut with values outside of [0..1];
```

Comment:
I thought that for "regular" images we gave colors channel values in the range of 0-255. Does the extended srb use 0-1?

Reply:
Yep, you have to use `toByteData(ImageByteFormat.rawExtendedRgba128)` to get normalized color components.  I can add a note here linking that.

Reply:
(added a note about ImageByteFormat.rawExtendedRgba128)